src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -236,6 +227,7 @@ class JibriSelenium(
             }
         }
         CallPage(chromeDriver).leave()
+        currCallUrl = null
```

Comment:
Is currCallUrl being stored for logging purposes?

Reply:
yup, only for that which is annoying, but i think it's valuable to have in the logs.